specs/proposals/blob-signing.md

```diff
@@ -0,0 +1,282 @@
+
+### Scenario 2: Blob signing and verification with registry-based distribution
+
+The platform team at Sarah’s company uses their existing OCI-compliant container registry for distributing SBOMs while maintaining the current generation process. The CI/CD pipeline continues to build binaries, generate SBOMs, and sign them as before. However, instead of uploading these artifacts to the company website, the pipeline now packages the SBOM and its signature as OCI artifacts and publishes them to the container registry. This ensures SBOMs and signatures are distributed in accordance with OCI standards across a multi-cloud environment. Additionally, the compliance team can verify the SBOMs directly from the registry, eliminating the need to download them to the filesystem. This streamlines the verification process and reduces the cost of downloading large SBOMs.
+
+## Proposal 
+
+This section outlines the proposed solution for using Notation CLI commands to sign and verify blobs. The following points are not covered in this document:
+- Detailed command usage, which will be covered in the individual command specifications at [Notation CLI Docs](https://github.com/notaryproject/notation/tree/v1.3.0/specs/commandline).
+- Distribution of certificates for verification across different scenarios.
+- Compatibility of the payload format. While some users are utilizing Notary Project-compliant signature envelopes, the payload format for blobs differs from the [Notary Project specification](https://github.com/notaryproject/specifications/blob/main/specs/signature-specification.md#blob-payload). 
+- CI/CD experience. This document specifically focuses on using Notation CLI commands.
+
+In general, all blob-related commands are grouped under the `notation blob` command for easy discovery and to avoid mixing them with OCI-based artifact commands.
+
+### Blob Sign and verify with file-based distribution
+
+For file-based distribution, such as SBOMs or release artifacts shared via a website or file transfer system, the process of signing and verifying blobs follows a similar flow. The steps below outline the user experience for signing and verifying a file, using an SBOM file named `sbom.json` as an example.
+
+**Prerequisites:**
+- SBOMs, release artifacts or other files are created and ready for publishing
+
+**Steps:**
+
+1. Sign a file on filesystem with a specified signature envelope:
+
+    ```shell
+    notation blob sign --signature-format cose sbom.json
+    ```
+
+    This generates a signature named `sbom.json.cose.sig` in the same directory as `sbom.json`. The signature file name follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/main/specs/signature-specification.md#blob-signatures).
```

Comment:
Please use a versioned link or a link with commit hash.

Reply:
It seems the blob spec was not released, so it cannot use versioned link.

Reply:
The blob spec was not released yet, so I cannot use a versioned link, will use a commit hash